ReportPortal.ATM/ReporterLibrary/Reporters/ExtentReporterService.cs

```diff
@@ -0,0 +1,48 @@
+﻿using AventStack.ExtentReports;
+using AventStack.ExtentReports.Reporter;
+using ReporterLibrary.Interfaces.Reporters;
+using System;
+using System.IO;
+using System.Reflection;
+
+namespace ReporterLibrary.Reporters
+{
+    public class ExtentReporterService : IReporterService
+    {
+        private readonly ExtentReports _extenReport;
+        private ExtentTest _extentTest;
+
+        public ExtentReporterService()
+        {
+            var path = Path.Combine(Path.GetDirectoryName(Assembly.GetExecutingAssembly().Location), "results'\'");
```

Comment:
You could make this more flexible by receiving a file path attribute in the constructor and, later in the test solution, you can get this from the configuration and whatnot.

Reply:
Changed to receive path as parameter